### PR TITLE
Add missing examples to Microsoft.Logic 

### PR DESCRIPTION
**Pull request by refortie**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/logic/resource-manager/Microsoft.Logic/stable/2016-06-01/examples/WorkflowsCreateOrUpdate.json [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+            }
+          }
+        }
+      },
+      "location": "brazilsouth",
+      "tags": {}
+    }
+  },
+  "responses": {
+    "200": {
```
Comment (jianghaolu) [inline specification/logic/resource-manager/Microsoft.Logic/stable/2016-06-01/examples/WorkflowsCreateOrUpdate.json]: Please also add an example response for 201.

File: specification/logic/resource-manager/Microsoft.Logic/stable/2016-06-01/examples/WorkflowsCreateOrUpdate.json [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+        "state": "Enabled",
+        "version": "08586677515911718341",
+        "accessEndpoint": "http://tempuri.org",
+        "integrationAccount": {
+          "name": "test-integration-account",
+          "id": "/subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/resourceGroups/test-resource-group/providers/Microsoft.Logic/integrationAccounts/test-integration-account",
+          "type": "Microsoft.Logic/integrationAccounts"
+        },
+        "definition": {
+          "$schema": "https://schema.management.azure.com/providers/Microsoft.Logic/schemas/2016-06-01/workflowdefinition.json#",
```
Comment (jianghaolu) [inline specification/logic/resource-manager/Microsoft.Logic/stable/2016-06-01/examples/WorkflowsCreateOrUpdate.json]: This link gives BlobNotFound. Can you confirm?

**Review by jianghaolu — CHANGES_REQUESTED** [review]

File: specification/logic/resource-manager/Microsoft.Logic/stable/2016-06-01/examples/WorkflowsCreateOrUpdate.json (hunk 2 above)
Comment (refortie) [reply specification/logic/resource-manager/Microsoft.Logic/stable/2016-06-01/examples/WorkflowsCreateOrUpdate.json]: That is correct.

File: specification/logic/resource-manager/Microsoft.Logic/stable/2016-06-01/examples/WorkflowsCreateOrUpdate.json (hunk 1 above)
Comment (refortie) [reply specification/logic/resource-manager/Microsoft.Logic/stable/2016-06-01/examples/WorkflowsCreateOrUpdate.json]: Added

**Review by jianghaolu — APPROVED** [review]

**Pull request merged**